src/ui/mainwindow.cpp

```diff
@@ -636,7 +636,7 @@ void MainWindow::setupTabBar()
     m_tabBar->setDrawBase(false);
     m_tabBar->setDocumentMode(true);
     m_tabBar->setElideMode(Qt::ElideRight);
-    m_tabBar->setStyleSheet(QStringLiteral("QTabBar::tab { width: 150px; }"));
+    m_tabBar->setStyleSheet(QStringLiteral("QTabBar::tab { width: 150px; height: 39px; }"));
```

Comment:
Setting tab height is a bad idea, as different themes have different tab heights.